### PR TITLE
[#145] implement new resolver actor

### LINKED ISSUE #145: Implement new resolver actor
implement rest resolver actor interface

### resolver actor function list:
- find mapped service(in static path) actor and send the request to it. If not exist, send not found response

### PR DESCRIPTION
**Pull request by ztkmkoo**
Closes #145

### CODE REVIEW

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/entity/DssDefaultRestRequest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Override
+    public DssRestMethodType getMethodType() {
+        return null;
+    }
+
+    @Override
+    public DssRestContentType getRestContentType() {
+        return null;
+    }
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/entity/DssDefaultRestRequest.java]: Need implement here

**Comment by ztkmkoo** [comment]
@Alencion @RulLu16 @Doyuni @yeonnseok 
I'll try to patch resolve actor.

Definition of ResolveActor => handling request from netty handler and pass the request to specific service actor and handling service actor's response to client side.

Feel free to review this. (I will merge it at Monday night)

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.Objects;
+
+/**
+ * @author Kebron ztkmkoo@gmail.com
+ * @create 2020-11-28 15:09
+ */
+public interface DssRestResolverCommand extends DssResolverCommand {
+
+    @Getter
+    class RestRequest extends DssRestCommand.RestRequest implements DssRestResolverCommand {
```
Comment (Doyuni) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java:17]: May I ask why define classes in interface?

Comment (ztkmkoo) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java:17]: Hi, (thank your review)

First of all, akka guide us define the command like follows
``` java
// from https://doc.akka.io/docs/akka/current/typed/actors.html#a-more-complex-example
static interface RoomCommand {}

public static final class GetSession implements RoomCommand {
  public final String screenName;
  public final ActorRef<SessionEvent> replyTo;

  public GetSession(String screenName, ActorRef<SessionEvent> replyTo) {
    this.screenName = screenName;
    this.replyTo = replyTo;
  }
}
```

And i think it is not a comman way,
so i declared the command interface and define the implement of the command in the class.

And when i define the class in the interface, I can access the `inner class` like DssRestResolverCommand.RestRequest, and it is much clear to me understand what kind of commands.
If I define in new .java file, I may need to make the class name like DssRestResolverRestRequestCommand. And it is so long.

Comment (ztkmkoo) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java:17]: @Doyuni

Comment (Doyuni) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestResolverCommand.java:17]: Thank you for answer :D 
I agree on this way, It's legible!

**Pull request merged**